src/main/java/uk/gov/companieshouse/efs/api/submissions/repository/SubmissionRepositoryImpl.java

```diff
@@ -111,12 +113,35 @@ public void updateBarcode(String id, String barcode) {
 
     @Override
     public List<Submission> findDelayedSubmissions(SubmissionStatus status, LocalDateTime before) {
-        LOGGER.debug(String.format("Fetching submissions with status: [%s] and before [%s]", status, before.toString()));
-        List<Submission> submissions = template
-            .find(Query.query(Criteria.where(STATUS).is(status).and(LAST_MODIFIED_AT).lte(before)), Submission.class,
-                SUBMISSIONS_COLLECTION);
-        LOGGER.debug(String.format("Found [%d] submissions with status: [%s] and before [%s]", submissions.size(), status,
-                before.toString()));
+        LOGGER.debug(
+            String.format("Fetching submissions with status: [%s] last modified before [%s]",
+                status, before.toString()));
+        List<Submission> submissions = template.find(Query.query(Criteria.where(STATUS)
+            .is(status)
+            .and(LAST_MODIFIED_AT)
+            .lte(before)
+            .not()
+            .and(FORM_TYPE)
+            .is("SH19_SAMEDAY")), Submission.class, SUBMISSIONS_COLLECTION);
```

Comment:
is this part right?
a SH19_SAMEDAY was picked up for me when running this.
![Screenshot 2021-07-12 at 14 49 24](https://user-images.githubusercontent.com/2736331/125298774-5f2a5300-e320-11eb-84bf-97c15d422bc2.png)

Reply:
Good spot, I think this is a bug

Reply:
I pushed a change to the query to fix the bug.

Reply:
that's better I see the appropriate SH19 in the appropriate email now.